packages/libs/react-ui/src/components/Notification/Notification.css.ts

```diff
@@ -0,0 +1,127 @@
+import { ColorType, sprinkles, vars } from '../../styles';
+
+import {
+  createVar,
+  fallbackVar,
+  style,
+  styleVariants,
+} from '@vanilla-extract/css';
+
+const contrastColor = createVar(),
+  surfaceColor = createVar();
+
+export const containerClass = style([
+  sprinkles({
+    borderWidth: '$md',
+    display: 'flex',
+    justifyContent: 'flex-start',
+    alignItems: 'flex-start',
+    width: 'max-content',
+    height: 'min-content',
+    position: 'relative',
+  }),
+  {
+    border: `1px solid ${fallbackVar(contrastColor, vars.colors.$neutral6)}`,
+    color: fallbackVar(contrastColor, vars.colors.$neutral6),
+    backgroundColor: fallbackVar(surfaceColor, vars.colors.$neutral2),
```

Comment:
Both of these you can still move to sprinkles, this way you won't have to use var!